### ISSUE TITLE
Help drawer lacks explicit focus trap and focus-return behavior

### ISSUE THREAD

**Issue by RobertAgterhuis** [RobertAgterhuis/myAgentic-IT-Project-team-V2#1534]
Implements #1531
